docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  integration_tests:
+    build:
+      dockerfile: Dockerfile
+      context: ${GSY_E_TARGET_REPO:-./}
+    container_name: ${CONTAINER_NAME:-gsy-e-tests}
+    entrypoint: ["gsy-e"]
+    command: ${COMMAND:-}
+    environment:
+      - REDIS_URL=${REDIS_URL:-redis://redis.container:6379/}
+    volumes:
+      - ${GSY_E_TARGET_REPO:-integration_tests}:/gsy-e
+
+  redis:
+    image: redis:6.2.5
+    restart: always
+    container_name: gsy-e-tests-redis
+    ports:
+      - "${PORT:-3679}:${PORT:-6379}"
+    hostname: ${HOST:-redis.container}
+    volumes:
+      - redis_volume:/data
+
+volumes:
+  integration_tests:
+    driver: local
+  redis_volume:
+    driver: local
```

Comment:
How come we need these volumes here ? I can understand for integration_tests (because I assume that this compose file will be used when running the integration tests, therefore it needs to read them from somewhere) however I do not think that a Redis volume will be something of use here.

Reply:
I thought of naming it because I saw that some hashed volumes were being left behind and they corresponded with the temporal volume the redis service creates. However these are only left behind if the normal execution of the integration tests is interrupted. At least I tend to run and stop it without necessarily waiting till everything is finished and therefore cleaned up.
By naming it, the redis service would always use the same volume, and on interruption, no hashed unused volumes would be left behind, as redis_volume would persist and be reused.